### PR TITLE
Update dataminer.services docs

### PR DESCRIPTION
**Pull request by AlexanderVK-SLC**
Improve the documentation for dataminer.services with updated information.

### CODE REVIEW

**Review by AntonPlanckeSLC — APPROVED** [review]

File: user-guide/Cloud_Platform/CloudAdminApp/Accessing_the_Admin_app.md [hunk 1]
```diff
@@ -4,9 +4,29 @@ uid: Accessing_the_Admin_app
...
 
-> [!TIP]
-> From DataMiner 10.2.9/10.3.0 onwards, you can navigate directly to the Admin app from DataMiner Cube by selecting *Cloud Admin* in the *Apps* pane.
+## Via dataminer.services
+
+You can also go to the app via dataminer.services:
+
+1. Go to [dataminer.services](https://dataminer.services)
+
+1. Find the DMS in the list and click the *Admin* icon.
```
Comment (MariekeGO) [inline user-guide/Cloud_Platform/CloudAdminApp/Accessing_the_Admin_app.md:21]: What about the Admin icon higher up under Apps? 
image

Perhaps we can give the user the choice here? 

```suggestion
1. If you want to open the Admin app to manage a specific DMS, find the DMS in the list and click the *Admin* icon.

   If you are not interested in a specific DMS, click the Admin icon under *Apps*.
```

File: user-guide/Cloud_Platform/Connecting_to_cloud/Connect_to_dataminer_services.md [hunk 2]
```diff
@@ -60,7 +60,7 @@ Before you connect your DMS to dataminer.services (unless you [connect with a DM
 
 1. Select the checkbox to agree to the terms of service and click *Connect*.
 
-1. On the System Center \> *Cloud* page, wait until the status under *Cloud info* changes to *Connected*. This can take up to half a minute.
+1. On the System Center \> *Cloud* page, wait until the status under *Registration status* changes to *Registered*. This can take up to half a minute.
```
Comment (MariekeGO) [inline user-guide/Cloud_Platform/Connecting_to_cloud/Connect_to_dataminer_services.md:63]: Is this change to *Registration status* implemented by RN 38715? We'll need to make sure the documentation is correct for people with older supported DataMiner versions as well, as there is only one version of the documentation for all DataMiner users.

Comment (AlexanderVK-SLC) [reply user-guide/Cloud_Platform/Connecting_to_cloud/Connect_to_dataminer_services.md:63]: Yes indeed, but the 'Cloud info' was already wrong, that doesn't exist right now either. 
It's definitely worth adding now that currently it says Status: Connected' when it's done depending on the version, but will indeed be changed to 'Registration status: Registered'.

File: user-guide/Cloud_Platform/CloudAdminApp/Accessing_the_Admin_app.md (hunk 1 above)
Comment (AlexanderVK-SLC) [reply user-guide/Cloud_Platform/CloudAdminApp/Accessing_the_Admin_app.md:21]: Yes, sounds good :) One opens the Organization overview, the other the DMS overview of the one selected indeed.

**Pull request merged**